Tests/ML/configs/other/test_hello_container.py

```diff
@@ -0,0 +1,86 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import torch
+import pytest
+from typing import Optional
+from InnerEye.Common import fixed_paths_for_tests
+from InnerEye.ML.configs.other.HelloContainer import HelloDataModule
+
+
+def test_cross_validation_splits() -> None:
+    '''
+    Test that the exemplar HelloDataModule correctly splits the data set for cross validation.
+
+    We test that unreasonable values raise an exception (i.e. a split index greater than the number of splits,
+    or requesting a number of splits larger than the available data).
+     - We test that between splits the train and validation data differ, and the test data remains the same.
+     - We test that all the data is used in each split.
+     - We test that across all the splits the validation data use all of the non-test data.
+    '''
+    # Get full data-set for comparison
+    root_folder = fixed_paths_for_tests.full_ml_test_data_path()
+    data_module_no_xval = HelloDataModule(root_folder=root_folder)
+
+    for number_of_cross_validation_splits in [0, 1, 5]:
```

Comment:
nitpicking - I think it should also fail for splits=1. Or are you treating that the same as splits=0?

Reply:
I'm treating it the same as 0, but I can swap it to fail if that's what ML people would expect @ant0nsc?

Reply:
That's fine, it's just a piece of example code